ansible/roles/kubernetes_irods_csi_driver/tasks/main.yml

```diff
@@ -99,7 +100,7 @@
   environment:
     KUBECONFIG: "{{ lookup('env', 'KUBECONFIG') }}"
   ansible.builtin.shell:
-    cmd: kubectl kustomize --enable-helm | kubectl create -f -
+    cmd: kubectl kustomize --enable-helm | kubectl apply -f -
```

Comment:
This is going to have to be updated so that it can either create or apply the manifests, but I haven't made that change yet.